README.md

```diff
@@ -189,6 +189,20 @@ We are [browserify](http://browserify.org/)able and [webpack](http://webpack.git
 npm install algoliasearch --save
 ```
 
+#### Typescript
```

Comment:
It would be good if this was showing in the TOC (table of content). And I would change the title to TypeScript typings